test/saucelabs/test.py

```diff
@@ -104,35 +122,27 @@ def parse_args(args):
             print 'Please try again...'
             sys.exit(1)
 
-    # if 3rd parameter equal 'true' use only top browser
-    caps_used = [top_browser] if singlebrowser else desired_cap_list
-
-    for current_desired_cap in caps_used:
+    for browser in browsers:
```

Comment:
not ideal to use the same namespace as one of your `argument`

Reply:
fixed